css/reset.css

```diff
@@ -1,5 +1,5 @@
 
-@import url('./utilities.css');
+@import url('./variables.css');
```

Comment:
you don't need to import any css file in any other file other than the `main.css`. Since all the stylesheets will be imported there, every CSS file will get connected with each other.

Reply:
ok i will do that.